examples/StandardFirmataEthernet/StandardFirmataEthernet.ino

```diff
@@ -103,17 +104,25 @@
 
 #if defined remote_ip && !defined remote_host
 #ifdef local_ip
-EthernetClientStream stream(client, local_ip, remote_ip, NULL, remote_port);
+EthernetClientStream stream(client, local_ip, remote_ip, NULL, network_port);
 #else
-EthernetClientStream stream(client, IPAddress(0, 0, 0, 0), remote_ip, NULL, remote_port);
+EthernetClientStream stream(client, IPAddress(0, 0, 0, 0), remote_ip, NULL, network_port);
 #endif
 #endif
 
 #if !defined remote_ip && defined remote_host
 #ifdef local_ip
-EthernetClientStream stream(client, local_ip, IPAddress(0, 0, 0, 0), remote_host, remote_port);
+EthernetClientStream stream(client, local_ip, IPAddress(0, 0, 0, 0), remote_host, network_port );
 #else
-EthernetClientStream stream(client, IPAddress(0, 0, 0, 0), IPAddress(0, 0, 0, 0), remote_host, remote_port);
+EthernetClientStream stream(client, IPAddress(0, 0, 0, 0), IPAddress(0, 0, 0, 0), remote_host, network_port);
+#endif
+#endif
+
+#if !defined remote_ip && !defined remote_host
+#ifdef local_ip
+EthernetServerStream stream(client, local_ip, network_port);
```

Comment:
Please remove the `client` parameter here and on line 125. Then make sure StandardFirmataEthernet compiles when configured for TCP server.